runtime/doc/lua.txt

```diff
@@ -301,7 +301,9 @@ arguments separated by " " (space) instead of "\t" (tab).
 							*:luafile*
 :[range]luafile {file}
 			Execute Lua script in {file}.
-			The whole argument is used as a single file name.
+                        The whole argument is used as the filename (like
+                        |:edit|), spaces do not need to be escaped.
+                        Alternatively you can |:source| Lua files.
```

Comment:
Probably should stick with tabs here to stay consistent.

Reply:
95% of the file uses spaces so i converted the whole file